roles/tas_single_node/defaults/main.yml

```diff
@@ -43,11 +46,6 @@ tas_single_node_trillian:
     host: trillian-mysql-pod
     port: 3306
 
-tas_single_node_config_dir: /etc/rhtas
-tas_single_node_certs_dir: "{{ tas_single_node_config_dir }}/certs"
-tas_single_node_kube_manifest_dir: "{{ tas_single_node_config_dir }}/manifests"
-tas_single_node_kube_configmap_dir: "{{ tas_single_node_config_dir }}/configs"
-
 tas_single_node_private_key_filename: rhtas.key
```

Comment:
I think these will need to change for the following release (we will probably do lists of certificates) and for the first release of the collection we do not promise exposing them. Let's move them all to vars - we'll expose in a future release when we're ready for customers to provide their own certs/keys.

Reply:
Ok will change this.